firebase-common/data-collection-tests/src/test/java/com/google/firebase/DataCollectionTestUtil.java

```diff
@@ -52,10 +52,17 @@ static SharedPreferences getSharedPreferences() {
             Context.MODE_PRIVATE);
```

Comment:
How does this test work if it's using user credential encrypted context?(i.e. not the direct boot one?)

Reply:
Its because of the config sdk=19